framework/include/utils/MooseMeshUtils.h

```diff
@@ -99,4 +99,16 @@ computeFaceInfoFaceCoord(FaceInfo & fi,
 {
   coordTransformFactor(fi.faceCentroid(), fi.faceCoord(), coord_type, rz_radial_coord);
 }
+
+/**
+ * Crate a new set of element-wise IDs by finding unique combinations of existing extra ID values
+ * @param mesh input mesh
```

Comment:
Need to have a short description on what algorithm is used for generating the new IDs.